src/tests/test_notifications.py

```diff
@@ -1,8 +1,8 @@
 """Testing Suite for all notifications."""
 from flask import current_app
 
-from api.utils.notifications.email_notices import send_email
-from tests.base_test import BaseTestCase
+from ..api.utils.notifications.email_notices import send_email
+from .base_test import BaseTestCase
```

Comment:
I had such an issue, I resolved it by moving 
```python
 from test.base_test import BaseTestCase
``` 
to be the first import on top of the file to avoid this relative imports. 😅 During my attempts to allow us to run test in a specific file rather than the whole test case I did introduce this. I will document that.

Reply:
Hehehe. The current order seems to work without issues. Let me try switching them out. I also tested that you run individual tests on all files now.

Reply:
Just confirmed @georgreen. The single file test fails when I revert to what was there. I think the lines you included in `base_test.py` to add to the src directory python path work well flawlessly relative imports so I'll leave the imports as is.